README.md

```diff
@@ -3,28 +3,9 @@
 
 Check out the [TCK Reference Guide](https://jakartaee.github.io/cdi-tck/) to get acquainted with the CDI TCK and learn how to execute and debug it.
 
-## Building CDI TCK artifacts
-To build the CDI TCK artifacts, use:
-
-`mvn -DtckAuditSourceGen install`
-
-or when compiling against staged Jakarta artifacts:
-
-`mvn -DtckAuditSourceGen -Pstaging install`
-
-## Building the CDI TCK distribution
-The CDI TCK distribution artifact is built by specifing an additional `-Drelease` property to build the TCK reference
-documentation and distribution bundle, e.g.:
-
-`mvn -DtckAuditSourceGen -Drelease install`
-
-## Eclipse Continuous Integration Environment
-The Eclipse continuous integration environment interface for the CDI project is located at https://ci.eclipse.org/cdi/
-The https://github.com/jakartaee/cdi/wiki/Eclipse-CI-Release-Jobs page describes the jobs found there.
-
```

Comment:
Should this huge section really be removed? It seems to me it contains some useful info. I'd just remove `-DtckAuditSourceGen` from the commands and everything else would still make sense I think?